templates/haproxy.cfg

```diff
@@ -4,7 +4,6 @@ global
     chroot /var/lib/haproxy
     stats socket /run/haproxy/admin.sock mode 660 level admin
     stats timeout 2m
-    server-state-file {{ LOAD_BALANCER_STATE_FILE }}
```

Comment:
@smarnach Is it harmful to have a state file that isn't loaded? E.g. if we kept the file to reflect HAProxy state, but just didn't load it with the line that was deleted below.

Reply:
I don't think the state file is very useful for debugging, and as you said, we can always create one if we think we need it.  Instead, we should make haproxy's stats web page available somehow, since it's quite useful.  And we should install the haproxy exporter for Prometheus, which I will be doing next sprint.

Reply:
Ah yes, the exporter will be plenty useful. Nevermind then.